README.md

```diff
@@ -1 +1,48 @@
-# __template
\ No newline at end of file
+![LUSID by Finbourne](https://content.finbourne.com/LUSID_repo.png)
+
+# ${REPO_NAME}
+
+## Getting Started
+
+## Contributing
+
+We welcome community participation in our tools. For information on contributing see our article [here](/finbourne/${REPO_NAME}/docs)
+
+## Reporting Issues
+If you encounter any issues please report these the Github [issues page](https://github.com/finbourne/${REPO_NAME}/issues).
+
+---
+
+# TODOs
+
+- [ ] update repo name in template
+- [ ] set up automated build
```

Comment:
as Github Actions?  with link to GA documentation?

Reply:
done